### PR TITLE
test docs + tag

### PR DESCRIPTION
**Pull request by snopoke**
Wanted to run all tests that use the `run_with_both_backends` decorator and found the nose attr plugin useful for it. Might be other uses too.

@dimagi/test-team

### CODE REVIEW

**Comment by snopoke** [comment]
(cancelled build)

**Review by czue — APPROVED** [review]
cool!

File: README.md [hunk 1]
```diff
@@ -336,6 +337,20 @@ To avoid having to run the databse setup for each test run you can specify the `
...
 See `corehq.tests.nose.HqdbContext` for full description of `REUSE_DB`.
 
+### Running tests by tag
+You can run all tests with a certain tag as follows:
+
+    $ ./manage.py test --attr=tag
+
+Available tags:
+
+  * dual_backend: all tests decorated with `run_with_all_backeds`
```
Comment (millerdev) [inline README.md]: nit: maybe rename this to `all_backends` to be consistent with the decorator name and in case we ever have more than two backends?

**Review by millerdev — COMMENTED** [review]
Nice!

File: README.md (hunk 1 above)
Comment (benrudolph) [inline README.md]: also nice: `run_with_all_backeds`

**Pull request merged**